pkg/ebpf/tracer.go

```diff
@@ -101,6 +101,8 @@ func NewFlowFetcher(cfg *FlowFetcherConfig) (*FlowFetcher, error) {
 	if enableRtt == 0 {
 		// Cannot set the size of map to be 0 so set it to 1.
 		spec.Maps[flowSequencesMap].MaxEntries = uint32(1)
+	} else {
+		log.Infof("RTT calculations are enabled")
```

Comment:
is this for debugging ? we can know what is enabled or not from looking at the configs env var correct ?

Reply:
Was just meant as info to see this via logs yeah...can be removed safely.

Reply:
only problem I see is that if both directions (Ingress and Egress are not enabled) RTT calculations will be disabled and message will be present only in logs.

Reply:
is it even possible to disable tc on one side but not the other ?

Reply:
well do you mean tc queues? I am not sure but don't think its possible. 
What I had meant is if ebpf agent somehow is set to attach only at one of Ingress or Egress, the [DIRECTION](https://github.com/netobserv/netobserv-ebpf-agent/blob/main/docs/config.md?plain=1#L44) configuration for agent. I believe right now this option is not exposed to the operator so we shouldn't worry too much.